rest-api-spec/src/main/resources/rest-api-spec/api/exists.json

```diff
@@ -42,6 +46,27 @@
         "routing": {
           "type" : "string",
           "description" : "Specific routing value"
+        },
+        "_source": {
+          "type" : "list",
+          "description" : "True or false to return the _source field or not, or a list of fields to return"
+        },
+        "_source_exclude": {
+          "type" : "list",
+          "description" : "A list of fields to exclude from the returned _source field"
+        },
+        "_source_include": {
+          "type" : "list",
+          "description" : "A list of fields to extract and return from the _source field"
+        },
```

Comment:
sure but what does it mean to get back e.g. 404 when I provide source_include? that the document is not there? or that the source is not there? or that the required fields are not there?